connector/fyle_integrations_platform_connector/apis/projects.py

```diff
@@ -23,7 +23,8 @@ def sync(self):
                     'attribute_type': self.attribute_type,
                     'display_name': self.attribute_type.replace('_', ' ').title(),
                     'value': project['name'],
-                    'source_id': project['id']
+                    'source_id': project['id'],
+                    'active': project['is_enabled']
                 })
 
             self.bulk_create_or_update_expense_attributes(project_attributes)
```

Comment:
`            self.bulk_create_or_update_expense_attributes(project_attributes, True)
`
you'll have to pass update flag with true so that the destination attribute will get updated